docs/ENGINE.md

```diff
@@ -0,0 +1,82 @@
+# Risk Engine
+
+## Overview
+The risk engine is responsible for handling and executing active invariants. It is the primary downstream consumer of ETL output. The risk engine will receive data from the ETL and execute the invariants associated with the data. If an invalidation occurs, the risk engine will return an `InvalidationOutcome` to the `EngineManager`. The `EngineManager` will then create an `Alert` using the `InvalidationOutcome` and publish it to the Alerting system.
+
+The Risk Engine will execute the invariants associated with some ingested input data and return an `InvalidationOutcome` to the `EngineManager`. The `EngineManager` will then create an `Alert` using the `InvalidationOutcome` and publish it to the Alerting system.
+
+The following diagram further exemplifies this key interaction:
+```mermaid
+graph LR;
```

Comment:
Rather than just having the event loop point to itself, you may be interested in expressing it as a loop: https://mermaid.js.org/syntax/sequenceDiagram.html#loops